### PR TITLE
Release prep

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

**Comment by VSEphpbb** [comment]
Updated and Fixes #20

**Comment by rxu** [comment]
Looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,27 +1,34 @@
-# phpBB Skeleton Extension
+# phpBB Skeleton Extension 
+
+phpBB's Skeleton Extension is a tool for extension authors to help speed up and simplify the task of starting a new extension project. It generates sample starter files and directories in a skeleton package that you can use to begin building your extension.
 
 ## Installation
 
-To install the source code from this repository, clone this extension repository to `phpBB/ext/phpbb/skeleton`:
+- [Download the latest release](releases).
```
Comment (3D-I) [inline README.md]: There aren’t any releases here

Comment (VSEphpbb) [inline README.md]: https://github.com/phpbb-extensions/phpbb-ext-skeleton/releases

**Comment by 3D-I** [comment]
There weren't.. :) :+1:

**Comment by michaelcullum** [comment]
<VSE> MichaelC rxu can you look at this last little PR, release prep https://github.com/phpbb-extensions/phpbb-ext-skeleton/pull/21
<VSE> please

Well I would but...

**Pull request merged**